### PR TITLE
Added throttle option to TerrainData.createMesh

### PR DESCRIPTION
**Pull request by IanLilleyT**
This PR originates from https://github.com/CesiumGS/cesium/pull/9286#issuecomment-756779332. It adds a `throttle` option to `TerrainData.createMesh` which says whether it should limit the number of asynchronous mesh creations. Note that `createMesh` now takes an options object instead of separate arguments.

For rendering purposes, throttling is enabled so that only the most relevant terrain tiles are processed, letting it quickly adapt to camera movements.

For `sampleTerrain`/`sampleTerrainMostDetailed`, throttling is disabled so that all heights are sampled at once. This ability is needed by https://github.com/CesiumGS/cesium/pull/9286 which calls `createMesh` when a `TerrainData` can't be interpolated from its raw buffer. Before, if the sample positions spanned a large number of tiles, many of the heights would be undefined because the system couldn't spawn a worker for every tile.

This PR also sets `TaskProcessor.maximumActiveTasks` constructor option to infinity by default.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @IanLilleyT!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+### 1.78 - 2021-02-01
+
+##### Additions :tada:
+
+- Added `Cartesian2.cross`. [#9305](https://github.com/CesiumGS/cesium/pull/9305)
```
Comment (IanLilleyT) [inline CHANGES.md:7]: Leftover from https://github.com/CesiumGS/cesium/pull/9305

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @IanLilleyT! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by lilleyse** [comment]
Looks good. Thanks @IanLilleyT

**Pull request merged**